tools/run-materialize.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+dl_materialized() {
+  INSTALL_DIR=$1
+  git -C "$INSTALL_DIR" clone git@github.com:MaterializeInc/materialize.git
+  cd "$INSTALL_DIR/materialize"
+}
+
+edit_rp_service() {
+  SERVICE_FILE_PY="$1/materialize/misc/python/materialize/mzcompose/services.py"
+  TARGET_VERSION=$2
+
+  # Grab the RP version from services.py
+  DEFAULT_RP_VERSION=$(grep "version: str" misc/python/materialize/mzcompose/services.py | cut -d'"' -f 2)
+  sed -i'' "s@$DEFAULT_RP_VERSION@$TARGET_VERSION@" "$SERVICE_FILE_PY"
```

Comment:
For somebody that knows `sed` this should be a one-liner. That person is not me.